server/src/main/java/com/lunchtime/service/MyUserDetailsService.java

```diff
@@ -0,0 +1,25 @@
+package com.lunchtime.service;
```

Comment:
Why did you save `implementation service` in this package?

Reply:
removed